### DISCUSSION TITLE
Start with Windows?

### DISCUSSION THREAD

**Discussion by Malygris-Six** [Heroic-Games-Launcher/HeroicGamesLauncher#2056] [Q&A]
This might be a stupid or obvious question, apologies if so, but is there an option to start Heroic when Windows boots? I thought I saw it in the 2.4.3 release (although I may have imagined it), but I recently upgraded to 2.5.0-b2 and I can't find it anywhere. But it seems like such an obvious feature, I can't help but feel that I have to be overlooking something. 

Thanks for help!

**Comment by letcus7** [comment]
Yes i can confirm this. No such an option. Seems like a necessary option for non-attended systems to auto start and download.

**Reply by regs01** [reply]
Copy shortcut file into %AppData%/Microsoft/Windows/Start Menu/Programs/Startup

**Reply by letcus7** [reply]
Yes, i already do this but a simple check box is more comfortable.
